css/layout.css

```diff
@@ -1,3 +1,81 @@
-* {
+/* Generl Rules/start */
+
+*,
+body,
+body * {
   outline: none;
+  box-sizing: border-box;
 }
+
+body {
+  max-width: 18.75rem;
+  margin: auto;
+}
+
+.social {
+  display: flex;
+  gap: 1rem;
+  align-items: center;
+  max-width: 2rem;
+}
+
+/* Generl Rules/end */
+
+/* Wrapper/start */
+
+.wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  padding: 0 1.3rem;
+  width: 100%;
```

Comment:
- [ ] [OPTIONAL] This is so fantastic to use CSS flexbox to layout your content, but there is just one more thing you can do to improve your code, that is DRY(don't repeat yourself). I can see that you have used the property `display: flex` at multiple places, please let's use DRY to keep the code clean by extracting the property into a separate class that can be reused at multiple places almost like what you did for the variables 👏 , see the example below: 

- CSS file
```
.d-flex {
  display: flex;
}

.flex-center {
  align-items: center;
  justify-content: center;
}
```
- HTML file
```
<div class="d-flex flex-center">{your content}</div>
```

Reply:
I like this one. That's an awesome tip. Really, thank you.<3